API/DTOs/UserDTO.cs

```diff
@@ -1,9 +1,42 @@
 namespace API.DTOs;
 
+/// <summary>
+/// Represents user account information
+/// </summary>
 public class UserDTO
 {
-    public int PlayerId { get; set; }
-    public DateTime? LastLogin { get; set; }
-    public DateTime Created { get; set; }
-    public string[] Scopes { get; set; } = [];
+    /// <summary>
+    /// Id of the user
+    /// </summary>
+    public int Id { get; set; }
+
+    /// <summary>
+    /// List of permissions granted to the user
+    /// </summary>
+    public string[]? Scopes { get; set; }
+
+    /// <summary>
+    /// Id of the associated player
+    /// </summary>
+    public int? PlayerId { get; set; }
+
+    /// <summary>
+    /// osu! id of the associated player
+    /// </summary>
+    public long? OsuId { get; set; }
+
+    /// <summary>
+    /// osu! country country code of the associated player
+    /// </summary>
+    public string? OsuCountry { get; set; }
+
+    /// <summary>
+    /// Preferred osu! mode of the associated player
+    /// </summary>
+    public int? OsuPlayMode { get; set; }
```

Comment:
This would be a new user setting (not player), but I think this property can be deleted since the frontend uses cookies for this at the moment.